template.json

```diff
@@ -78,6 +78,14 @@
             "walk": 30,
             "units": "ft",
             "hover": false
+          },
+          "encumbrance":{
+            "value" : 0,
+            "pct" : 0,
+            "max": 100,
+            "capacityMultiplyer" : 1,
+            "capacityBonus" : 0,
+            "encumbered" : false
```

Comment:
So does supporting formulas mean that overrides are the only viable way to edit that, and thus only one effect will be able to modify it? 

Is there a reason you keep putting capacity changes in the special traits menu rather than allowing items to modify them? I'm thinking of things like the Barbarian, Path of the Totem Warrior, Aspect of the Beast, Bear PHB page 50, of course that is a multiplier which may still be implemented here, though thinking on it this morning I may have implemented it incorrectly and thought it was right because I only tested a narrow case.

Reply:
You can also use the `add` active effect mode with formulas, which just appends your new data onto the end of the formula.

Items can use active effects to change anything within the special traits menu, that is just a place to put the input so users can modify it without active effects. For the Bear totem you would just create an effect that does something link `data.attributes.encumbrance.multiplier` | Multiply | `2`, which would stack on top of any existing modifier set manually in the special traits menu.